doc/api/readline.md

```diff
@@ -1313,6 +1313,16 @@ const { createInterface } = require('readline');
     <td>Delete from the current position to the end of line</td>
     <td></td>
   </tr>
+  <tr>
+    <td><kbd>Ctrl</kbd>+<kbd>Y</kbd></td>
+    <td>Yank the previously deleted text</td>
```

Comment:
Might be worthwhile describing what "yank" means.

Reply:
Done